dist/openshift/cincinnati.yaml

```diff
@@ -278,3 +278,6 @@ parameters:
   - name: PE_LOG_VERBOSITY
     value: "vv"
     displayName: Policy engine log verbosity
+  - name: GB_CINCINNATI_REPO
```

Comment:
Took me a bit to look down at `displayName` in order to unpack `GB` to "graph builder".  I see we already have `PE` for "policy engine".  I'd like to expand both to avoid abbreviations, but we can do that in follow up work if you don't want to pick it up here.

Reply:
That would require updating existing params - and iirc these are used as is in internal deployment. Can be done in a new PR